src/private/CABasicAnimation+MotionAnimator.h

```diff
@@ -24,7 +24,13 @@
 FOUNDATION_EXPORT
 CABasicAnimation *MDMAnimationFromTiming(MDMMotionTiming timing, CGFloat timeScaleFactor);
 
-// Attemps to configure the provided animation to be additive.
-// Not all animation value types are supported. If an animation's value type was not supported,
-// the animation will not be modified.
-FOUNDATION_EXPORT void MDMMakeAnimationAdditive(CABasicAnimation *animation);
+// Attempts to configure the provided animation to be additive and, if the animation is a spring
+// animation, will extract the initial velocity from the timing and apply it to the animation.
+//
+// Not all animation value types support being additive. If an animation's value type was not
+// supported, the animation's values will not be modified.
+//
+// This method assumes that the animation's from and to value do not match.
```

Comment:
I think it may be better to document what happens if they *do match*. Does the code crash, behavior is undefined (but non-crashing), or "nothing happens"?

Reply:
Done.